source/mir/ndslice/slice.d

```diff
@@ -1916,7 +1922,14 @@ public:
         }
         import mir.algorithm.iteration : equal;
         static if (__traits(compiles, this.lightScope))
-            return equal(this.lightScope, rslice.lightScope);
+        {
+            static if(this.L != rslice.L)
+                return false;
+            foreach(i; Iota!(this.L))
```

Comment:
`foreach(i; Iota!(min(this.L, rhs.L)))` where `min` is from mir.utility.

Reply:
Addressed in 4c97c4